src/API/OnboardingTasks.php

```diff
@@ -610,4 +641,230 @@ public function get_status() {
 
 		return rest_ensure_response( $status );
 	}
+
+	/**
+	 * Get the onboarding tasks.
+	 *
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function get_tasks() {
+		$profiler_data       = get_option( Onboarding::PROFILE_DATA_OPTION, array() );
+		$installed_plugins   = PluginsHelper::get_installed_plugin_slugs();
+		$product_types       = $profiler_data['product_types'];
+		$purchase_products   = $profiler_data['product_types']; // @todo This should use the product names.
+		$business_extensions = $profiler_data['business_extensions'];
+		$product_query       = new \WC_Product_Query(
+			array(
+				'limit'  => 1,
+				'return' => 'ids',
+				'status' => array( 'publish' ),
+			)
+		);
+		$products            = $product_query->get_products();
+		$wc_pay_is_connected = false;
+		if ( class_exists( '\WC_Payments' ) ) {
+			$wc_payments_gateway = \WC_Payments::get_gateway();
+			$wc_pay_is_connected = method_exists( $wc_payments_gateway, 'is_connected' )
+				? $wc_payments_gateway->is_connected()
+				: false;
+		}
+		$gateways                = WC()->payment_gateways->get_available_payment_gateways();
+		$enabled_gateways        = array_filter(
+			$gateways,
+			function( $gateway ) {
+				return 'yes' === $gateway->enabled;
+			}
+		);
+		$can_use_automated_taxes = ! class_exists( 'WC_Taxjar' ) &&
+			in_array( WC()->countries->get_base_country(), OnboardingTasksFeature::get_automated_tax_supported_countries(), true );
+
+		$marketing_extension_bundles        = RemoteFreeExtensions::get_extensions(
+			array(
+				'reach',
+				'grow',
+			)
+		);
+		$has_installed_marketing_extensions = array_reduce(
+			$marketing_extension_bundles,
+			function( $has_installed, $bundle ) {
+				if ( $has_installed ) {
+					return true;
+				}
+				foreach ( $bundle['plugins'] as $plugin ) {
+					if ( $plugin->is_installed ) {
+						return true;
+					}
+				}
+				return false;
+			},
+			false
+		);
+
+		$tasks = array(
+			array(
+				'id'         => 'setup',
+				'isComplete' => get_option( 'woocommerce_task_list_complete' ) === 'yes',
+				'isHidden'   => get_option( 'woocommerce_task_list_hidden' ) === 'yes',
+				'title'      => __( 'Get ready to start selling', 'woocommerce-admin' ),
+				'tasks'      => array(
+					array(
+						'id'          => 'store_details',
+						'title'       => __( 'Store details', 'woocommerce-admin' ),
+						'content'     => __(
+							'Your store address is required to set the origin country for shipping, currencies, and payment options.',
+							'woocommerce-admin'
+						),
+						'actionLabel' => __( "Let's go", 'woocommerce-admin' ),
+						'actionUrl'   => '/setup-wizard',
+						'isComplete'  => isset( $profiler_data['completed'] ) && true === $profiler_data['completed'],
+						'isVisible'   => true,
+						'time'        => __( '4 minutes', 'woocommerce-admin' ),
+					),
+					array(
+						'id'            => 'purchase',
```

Comment:
This task is currently rendering as "Add physical to my store" for me; I see you have a couple todo's around this, but thought I'd mention. I'm not precisely sure what this task is intended for but it seems that it'll be handled in another PR?

Reply:
I think it's best to go ahead and handle here.  Pushed some updates in https://github.com/woocommerce/woocommerce-admin/pull/7512/commits/859fe940ebfc05d731fa774742055a6eead74ba8.